projects/daq2/kc705/system_bd.tcl

```diff
@@ -1,15 +1,16 @@
+## Offload attributes
+set adc_offload_type 0
+set adc_offload_size [expr 510 * 1024]
 
-## FIFO depth is 4Mb - 250k samples
-set adc_fifo_address_width 16
+set dac_offload_type 0
+set dac_offload_size [expr 510 * 1024]
```

Comment:
On my machine I was able to build with 512*1024, both ADC and DAC. 
The utilization was 130 BRAM Tiles per FIFO, so 260 out of 445 available with a total 69% utilization per full project